sql/database.sql

```diff
@@ -12716,3 +12716,14 @@ PRIMARY KEY (`id`),
 KEY `fk_form_id` (`form_id`),
 KEY `fk_list_options_id` (`interpretation_list_id`, `interpretation_option_id`)
 ) ENGINE=InnoDB COMMENT='Detailed information of each vital_forms observation column';
+
+DROP TABLE IF EXISTS `jwt`;
+CREATE TABLE `jwt_grant_history` (
+`id` INT NOT NULL AUTO_INCREMENT
+ , `jti` VARCHAR(100) NOT NULL COMMENT 'Unique JWT id'
+ , `client_id` VARCHAR(80) NOT NULL COMMENT 'FK oauth2_clients.client_id'
+ , `jti_exp` TIMESTAMP NOT NULL COMMENT 'jwt exp claim when the jwt expires'
```

Comment:
would default date/time stuff to NULL unless know it will get filled (like the field below)